src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/EventLoggingService.kt

```diff
@@ -16,25 +20,50 @@ class EventLoggingService(
   private val objectMapper: ObjectMapper,
 ) {
 
+  fun snapshotPersonEntity(personEntity: PersonEntity) = snapshotEntity<PersonEntity>(personEntity)
+
+  fun snapshotPersonKeyEntity(personKeyEntity: PersonKeyEntity) = snapshotEntity<PersonKeyEntity>(personKeyEntity)
+
   fun recordEventLog(
-    beforePerson: Person?,
-    processedPerson: Person?,
-    uuid: String? = null,
+    beforePersonEntity: PersonEntity?,
+    afterPersonEntity: PersonEntity?,
```

Comment:
I tried this and found that they were often one and the same - maybe things have moved on since then

Reply:
This naming is definitely better